### PR TITLE
Ms option added

### PR DESCRIPTION
**Pull request by AakankshaShah**
## Related Issue 

Ms. option in teacher's profile form was missing. 
Closes: #[105]

### Describe the changes you've made

I added Ms. option in create  & update profile form.

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Enhancement in the existing features
- [ ] Other

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly whereever it was hard to understand.
- [x] My changes generate no new warnings.
- [x] I have verified/tested my code by running it locally.

## Additional Info (optional)
image

### CODE REVIEW

**Comment by Namanl2001** [comment]
Changes need to be done in the backend too

**Comment by AakankshaShah** [comment]
@Namanl2001  Sir there is no need to make changes in backend ,all changes have already been incorporated as much as I am aware.

**Comment by Namanl2001** [comment]
Yeah, you're right no changes required in the backend. But there are so many unintentional changes would be nice if you can resolve them so that we can have this or merged it to master. thanks

**Comment by AakankshaShah** [comment]
@Namanl2001  ok sir I will get it done.

**Comment by AakankshaShah** [comment]
@Namanl2001  Sir i have done the required changes.Please have a look and merge it accordingly.

File: backend/package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "helmet": "^4.4.1",
     "mongoose": "^5.9.20",
     "nodemailer": "^6.4.11",
+    "nodemon": "^2.0.7",
```
Comment (Namanl2001) [inline backend/package.json:22]: not needed

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,16355 @@
+{
+  "name": "project",
+  "version": "0.1.0",
+  "lockfileVersion": 1,
```
Comment (Namanl2001) [inline package-lock.json:4]: this project uses yarn...refer readme.md

File: src/components/form.jsx [hunk 3]
```diff
@@ -130,9 +129,7 @@ class form extends Component {
                       onChange={this.handleChange}
                       value={this.state.subject}
                     >
-                      <option defaultValue hidden>
```
Comment (Namanl2001) [inline src/components/form.jsx:133]: please don't remove "hidden"

File: src/components/form.jsx [hunk 4]
```diff
@@ -94,11 +94,10 @@ class form extends Component {
                       onChange={this.handleChange}
                       value={this.state.title}
                     >
-                      <option defaultValue hidden>
-                        Open this to select title
-                      </option>
+                      <option defaultValue>Open this to select title</option>
```
Comment (Namanl2001) [inline src/components/form.jsx:97]: include hidden

File: src/components/Footer.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import './style.css';
+
+function Footer() {
+  return (
+    <div className='main-footer'>
```
Comment (Namanl2001) [inline src/components/Footer.js:6]: this change seems unrelated

**Review by Namanl2001 — CHANGES_REQUESTED** [review]
small nits

File: package-lock.json (hunk 2 above)
Comment (AakankshaShah) [reply package-lock.json:4]: We are also using yarn .

File: src/components/Footer.js (hunk 5 above)
Comment (AakankshaShah) [reply src/components/Footer.js:6]: Sir this was not done by us ,while managing conflicts we accepted the changes.

File: package-lock.json (hunk 2 above)
Comment (Namanl2001) [reply package-lock.json:4]: this file is only generated when we use npm to install the dependencies

File: src/components/Footer.js (hunk 5 above)
Comment (Namanl2001) [reply src/components/Footer.js:6]: Rebase your branch then. Thanks

**Comment by AakankshaShah** [comment]
I have craeted a new PR #137  please review that one and close this one @Namanl2001 .
